packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -74,6 +76,19 @@ const COLUMN_STATE = {
   ),
   name: _('remoteState'),
 }
+const COLUMN_DISK = {
+  itemRenderer: (remote, { formatMessage }) => (
+    <span>
+      {remote.info &&
+        remote.info.used !== undefined &&
+        remote.info.available !== undefined &&
+        `${formatSize(remote.info.used)} / ${formatSize(
```

Comment:
It should be clear for the user if what you're showing is the free space or the used space.